AlexanderOtavka/ride-board
# Signup

Resolves #35 

# Changes

- User can now sign up leaving the password blank. A confirmation email will be sent to the user's email with a confirmation link. User will need to click on the confirmation link and set a password for the next sign up. 

# Manual Testing

1. Sign up. 
2. Wait for the email :"> 
3. Upon receiving the email, click on the "Confirm my account" link. 
4. Pray that it takes you to a page that lets you set your password. 

# Todo: 
1. [Page to set passwords](app/views/devise/confirmations/show.html.erb) needs some UI work.
2. Make it obvious to users that they can leave their password blank for now when signing up. (maybe hide the password field at first and have a button like "set my password" to make the password field pops up).
